validator.go

```diff
@@ -87,7 +87,7 @@ func newValidatorImpl(domains []string, usersFile string,
 	}
 
 	validator := func(email string) (valid bool) {
-		if email == "" {
+		if !allowAll && email == "" {
```

Comment:
I'd be tempted to make this more explicit, and move the block from below to the top of here

Eg 

```
validator := func(email string) (valid bool) {
  if allowAll {
    return true
  }

  // rest of function as is, removing the same if statement from below
```

Reply:
Will introduce that change in a minute

Reply:
Suggestion applied